README.md

```diff
@@ -29,9 +35,10 @@ After installation, make sure to commit the `.ddev` directory to version control
 
 ## Usage
 
-- Your project is now ready to run FunctionalJavascript and [Nightwatch](https://www.drupal.org/docs/automated-testing/javascript-testing-using-nightwatch) tests from Drupal core, or [Drupal Test Traits](https://gitlab.com/weitzman/drupal-test-traits) (DTT). All these types are tested in this repo. Some examples to try:
+- Your project is now ready to run [Functional](https://mglaman.dev/blog/do-you-need-functional-test), FunctionalJavascript and [Nightwatch](https://www.drupal.org/docs/automated-testing/javascript-testing-using-nightwatch) tests from Drupal core, or [Drupal Test Traits](https://git.drupalcode.org/project/dtt) (DTT). All these types are tested in this repo. Some examples to try:
```

Comment:
We dont yet test Functional in this repo.

Reply:
Great catch, I have tried to clarify this.